src/styles/blocks/home.scss

```
@@ -0,0 +1,393 @@
+.home {
+// display: flex;
+// justify-content: space-around;
+background-color: #F4F9E2;
+display: grid;
+// padding: 0 179px;
+justify-content: space-around;
+
+
+
+
+}
+
+.section__home {
```

```suggestion
.section {
  &__home {
```
Где ты описываешь класс `.section`? Что он делает?